src/harddisk/pata.rs

```diff
@@ -1,8 +1,32 @@
-use crate::svec::SVec;
+use core::sync::atomic::AtomicBool;
 
+use x86_64::instructions::port::{Port, PortReadOnly, PortWriteOnly};
+
+use crate::{nop, svec::SVec};
```

Comment:
`crate::nop` doesn't exist in `crate`

Reply:
Not *anymore*... and never in this repo. I'll run `rustfmt`as well, while I'm at it.